src/main/static/assets/templates/page_home.jade

```diff
@@ -32,14 +32,17 @@ microprofileio-header.home
   .card.text-image.flipped.pdf
     div
       div
-        h2 Eclipse MicroProfile 1.3 is now available!
-        p Version 1.3 consists of MicroProfile 1.2 plus the following updates: Config 1.2, Metrics 1.1, OpenAPI 1.0, OpenTracing 1.0, REST Client 1.0.
-        p Download the specification PDF to learn more about the architecture and required APIs.
-        a(href="https://github.com/eclipse/microprofile-bom/releases/download/1.3/microprofile-spec-1.3.pdf", target="_blank") Download PDF
+        h2 Eclipse MicroProfile 1.4 & 2.0 is now available!
```

Comment:
If my English is correct, this should be "are now available", since it signifies a plural form.

The & sign is coloured red. I don't exactly know why Github does this, but it could be an encoding warning?

Reply:
Thank you @ederks85 , I opened https://github.com/microprofile/microprofile-site/pull/79 for this fix.